lib/handbook/addon/docs/components/schema-block-group-renderer/-components/read-only/template.hbs

```diff
@@ -0,0 +1,17 @@
+<DocsDemo as |demo|>
+    <demo.example @name='schema-block-group-renderer.read-only.hbs'>
+        {{#each @schemaBlockGroups as |group|}}
+            <Registries::SchemaBlockGroupRenderer
+                @schemaBlockGroup={{group}}
+                @node={{@node}}
```

Comment:
is it possible to move `@node` to the component helper as well since not every schema-block-renderer needs a node?